test/unit_tests/js/modules/util/breakpoint-state-spec.js

```diff
@@ -57,8 +57,9 @@ describe( 'breakpoint-state', () => {
   describe( '.isInDesktop()', () => {
     it( 'should determine whether inside desktop breakpoint threshold', () => {
       expect( breakpointState.isInDesktop() ).toBe( true );
-      // TODO: Mock breakpointState.get() to a smaller size so this has false.
-      // expect( breakpointState.isInDesktop() ).toBe( false );
+
+      /* TODO: Mock breakpointState.get() to a smaller size so this has false.
```

Comment:
Speaking of typos, is this sentence grammatically correct?

Reply:
Good point. I know what it means, but I don't. Re-wrote it.